src/views/PlayView.vue

```diff
@@ -2,24 +2,38 @@
   <div class="purpleBackground">
     <div class="pageContainer">
       <div class="back">
-        <button class="rounded-button" @click="toggleBack">Back</button>
+        <button class="rounded-button" @click="toggleBack" aria-label="Go back to landing page">
```

Comment:
some buttons have attributes on new lines, but this one has all on one. It's best to try to stick with 1 format - but this is just code styling.

Reply:
formatting fixed